File: plugins/devkit/devkit-java-tests/testData/inspections/serializableCtor/SerializableClassButPropertyMappingAnnotationNotAvailable.java
import java.io.Serializable;

public class SerializableClassButPropertyMappingAnnotationNotAvailable implements Serializable {

  private static final long serialVersionUID = 1L;

  final String myString;
  final Integer myInteger;
  final Boolean myBoolean;

  public SerializableClassButPropertyMappingAnnotationNotAvailable(String string, Integer integer, Boolean bool) {
    myString = string;
    myInteger = integer;
    myBoolean = bool;
  }
}

File: plugins/devkit/devkit-kotlin-tests/testData/inspections/serializableCtor/SerializableClassButPropertyMappingAnnotationNotAvailable.kt
import java.io.Serializable

class SerializableClassButPropertyMappingAnnotationNotAvailable(val myString: String,
                                                                val myInteger: Int,
                                                                val myBoolean: Boolean) : Serializable {
  companion object {
    private const val serialVersionUID = 1L
  }
}
